examples/IRMQTTServer/IRMQTTServer.ino

```diff
@@ -2591,6 +2593,7 @@ void sendMQTTDiscovery(const char *topic) {
 #endif  // MQTT_ENABLE
 
 void loop(void) {
+  mdns.update();
```

Comment:
Needs the `#if MDNS_ENABLE` guard around the `update`

Reply:
Also Travis fails with the following when compiling for ESP32:
```
/home/travis/build/crankyoldgit/IRremoteESP8266/examples/IRServer/IRServer.ino:153:8: error: 'class MDNSResponder' has no member named 'update'
   mdns.update();
```

Reply:
OK will do. I prematurely pushed but will push a comprehensive commit.